### PR TITLE
Update background samples

### PR DESCRIPTION
**Pull request by kmohrman**
This branch updates the JSONs for the background samples to point to v9 samples where available. I've also added comments to the `make_jsons.py` script to indicate which samples are still temporary stand-ins (due to not being UL, or not being v9, or if they are from a "Pilot" run and have low statistics). We will need to continue to check DAS, replace the stand-ins with the correct samples and once they're available. 

I've also updated our summary [spreadsheet](https://docs.google.com/spreadsheets/d/16ZtsFffwNZDRUR5d0JcKBwI4UjG3pPhonaKZyFPamvs/edit#gid=0), adding a column to indicate whether or not the samples are v9.

### CODE REVIEW

File: analysis/topEFT/make_jsons.py [hunk 1]
```diff
@@ -476,122 +476,92 @@
...
         "xsecName": "WWTo2L2Nu",
     },
-
     "UL17_WWW_4F" : {
-        "path" : "/WWW_4F_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
-        "histAxisName": "WWW_centralUL17",
-        "xsecName": "WWW",
-    },
-
-    "UL17_WWW_4F_ext" : {
```
Comment (bryates) [inline analysis/topEFT/make_jsons.py:551]: Is there no longer a `WWW_ext` sample?

File: analysis/topEFT/make_jsons.py [hunk 2]
```diff
@@ -476,122 +476,92 @@
...
     },
-
     "UL17_ZZZ" : {
-        "path" : "/ZZZ_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8-v1/NANOAODSIM",
-        "histAxisName": "ZZZ_centralUL17",
-        "xsecName": "ZZZ",
-    },
-
-    "UL17_ZZZ_ext" : {
-        "path" : "/ZZZ_TuneCP5_13TeV-amcatnlo-pythia8/RunIISummer20UL17NanoAODv2-106X_mc2017_realistic_v8_ext1-v1/NANOAODSIM",
```
Comment (bryates) [inline analysis/topEFT/make_jsons.py:594]: Same question.

**Review by bryates — COMMENTED** [review]
Thanks @kmohrman for taking care of these. I noticed the `WWW_ext` and `ZZZ_ext` samples were removed. Were these merged together with the main samples in `nanoAODv9`?
Other than that, I have no objections to merging this PR.

File: analysis/topEFT/make_jsons.py (hunk 1 above)
Comment (kmohrman) [reply analysis/topEFT/make_jsons.py:551]: Thanks for pointing this out, I probably should have mentioned it in the PR's description. I replaced this WWW sample with a v9 one, and there does not seem to be an v9 "ext" sample available (maybe it's been merged with the non ext one as you mentioned, or maybe they just have not made an "ext" one yet).

File: analysis/topEFT/make_jsons.py (hunk 2 above)
Comment (kmohrman) [reply analysis/topEFT/make_jsons.py:594]: (Same answer as above.)

File: analysis/topEFT/make_jsons.py (hunk 1 above)
Comment (bryates) [reply analysis/topEFT/make_jsons.py:551]: Ok thanks. We should keep an eye on these. Looking at the `nEvents` entries, `WWW_4F` has 1.7M events (didn't change with v9) and `WWW_4F_ext` had another ~9M events.

Comment (kmohrman) [reply analysis/topEFT/make_jsons.py:551]: Ah, good point. I've added comments to these lines to remind us to check to see if "ext" samples become available.

Comment (bryates) [reply analysis/topEFT/make_jsons.py:551]: Thanks?

**Pull request merged**